lib/common/common/Jobs.inl

```diff
@@ -52,7 +52,7 @@ namespace JsUtil
     template<class TJobManager, class TJobHolder>
     void JobProcessor::PrioritizeJobAndWait(TJobManager *const manager, const TJobHolder holder, void* function)
     {
-        TemplateParameter::SameOrDerivedFrom<TJobManager, WaitableJobManager>;
+        TemplateParameter::SameOrDerivedFrom<TJobManager, WaitableJobManager>();
```

Comment:
The old code fails on clang. It was only a class name. The changed one calls the constructor.